controllers/component_controller.go

```diff
@@ -326,7 +326,7 @@ func (r *ComponentReconciler) Reconcile(ctx context.Context, req ctrl.Request) (
 	}
 
 	log.Info(fmt.Sprintf("Finished reconcile loop for %v", req.NamespacedName))
-	return ctrl.Result{}, err
+	return ctrl.Result{}, nil
```

Comment:
I noticed we're returning err on `generateGitops` in this controller, any reason why we would want to retry if those fail?

Reply:
Oops, that shouldn't be there.